json/images.json

```diff
@@ -0,0 +1,6 @@
+[
+    "sunset.jpg",
+    "bedroom.png",
+    "garden.jpeg",
+    "kitchen.jpg"
```

Comment:
we shouldn't use indices for image ids, this will fail really easily if we get the images out of order. let's do something like how you did it in characters.json

Reply:
unless we are just using file paths, then we don't need this file at all